ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.view.js

```diff
@@ -13,210 +13,316 @@
-            this.checkIsInWorkspace();
-            this.checkIfBlacklisted();
-            this.checkIfDownloadable();
-        },
-        onBeforeShow: function(){
-            this.resultActions.show(PopoutView.createSimpleDropdown({
-                componentToShow: MetacardInteractionsView,
-                dropdownCompanionBehaviors: {
-                    navigation: {}
-                },
-                modelForComponent: new Backbone.Collection([this.model]),
-                leftIcon: 'fa fa-ellipsis-v'
-            }));
-            this.resultIndicator.show(new ResultIndicatorView({
-                model: this.model
-            }));
-            this.resultAdd.show(PopoutView.createSimpleDropdown({
-                componentToShow: ResultAddView,
-                modelForComponent: new Backbone.Collection([this.model]),
-                leftIcon: 'fa fa-plus'
-            }));
-            this.handleResultThumbnail();
-        },
-        handleResultThumbnail: function() {
-            if (this.model.get('metacard').get('properties').get('thumbnail')) {
-                this.resultThumbnail.show(new HoverPreviewDropdown({
-                    model: new DropdownModel(),
-                    modelForComponent: this.model
-                }));
-            }
-        },
-        addConfiguredResultProperties: function(result){
-            result.showSource = false;
-            result.customDetail = [];
-            if (properties.resultShow) {
-                properties.resultShow.forEach(function (additionProperty) {
-                    if (additionProperty === 'source-id') {
-                        result.showSource = true;
-                        return;
-                    }
-                    var value = result.metacard.properties[additionProperty];
-                    if (value && metacardDefinitions.metacardTypes[additionProperty]) {
-                        switch (metacardDefinitions.metacardTypes[additionProperty].type) {
-                            case 'DATE':
-                                if (value.constructor === Array) {
-                                    value = value.map(function (val) {
-                                        return Common.getMomentDate(val);
-                                    });
-                                } else {
-                                    value = Common.getMomentDate(value);
+module.exports = Marionette.LayoutView.extend({
+    template(data) {
+        return (
+            <React.Fragment>
+                <div className="result-container" data-metacard-id={data.id}
+                data-query-id={data.metacard.queryId}>
+                <div className="container-indicator">
+                </div>
+                <div className="container-content">
+                    <div className="content-header">
+                        <span className="header-icon fa fa-history" title="Type: Revision" data-help="Indicates the type of result
```

Comment:
❗️ the help text for all of these looks like incorrect - accidental copy/paste?

Reply:
It's at least consistent with how they were before
https://github.com/codice/ddf/pull/3489/files/9c6e46023f51cff34d4234ebd9bfd1b6b18f168f#diff-bf0dcbdee8a22ca2b57876a2fa07e3a0L20

They could probably be better, but it's not a regression

Reply:
I'll downgrade my comment to a nit since it was existing